Modules/Core/Common/src/itkRealTimeClock.cxx

```diff
@@ -35,7 +35,7 @@ RealTimeClock::RealTimeClock()
   LARGE_INTEGER frequency;
   ::QueryPerformanceFrequency(&frequency);
 
-  this->m_Frequency = static_cast<FrequencyType>(static_cast<__int64>(frequency.QuadPart));
+  m_Frequency = static_cast<FrequencyType>(frequency.QuadPart);
```

Comment:
`this->` is sometimes required by GCC/clang, even if it compiles with MSVC.

Reply:
> `this->` is sometimes required by GCC/clang, even if it compiles with MSVC.

Thanks for your comment @dzenanz As far as I know, `this->` is only required for a class _template_, that is accessing a data member of one of its base classes. But please correct me if I'm wrong.

`RealTimeClock` is not a template, and it does not have a base class either. It just accesses its own data members. So I think all `this->` may be removed here.

Reply:
And CI has succeeded, proving your point 😄